modules/auxiliary/scanner/http/huawei_hg255s_directory_traversal.rb

```diff
@@ -0,0 +1,59 @@
+##
+# This file is part of the Metasploit Framework and may be subject to
+# redistribution and commercial restrictions. Please see the Metasploit
+# web site for more information on licensing and terms of use.
+#
+##
+
+require ‘msf/core’
+
+class Metasploit4 < Msf::Auxiliary
+   include Msf::Exploit::Remote::HttpClient
+
+   def initialize
+       super(
+           ‘Name’        => ‘Server Directory Traversal at Huawei HG255s’,
```

Comment:
This does not fit the naming convention.

```suggestion
           'Name'        => 'Huawei HG255 Directory Traversal',
```